src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -452,4 +452,7 @@
     <value>usage: NuGet locals &lt;all | http-cache | global-packages | temp&gt; [--clear | -c | --list | -l]
 For more information, visit http://docs.nuget.org/docs/reference/command-line-reference</value>
   </data>
+  <data name="Error_UnableToLocateBuildOutput" xml:space="preserve">
+    <value>No build found in {0}. Use the -Build option or build the project.</value>
```

Comment:
Add a comment describing the parameter. Needed for a translator doing the localization.
```xml
<comment>{0} is the package identity.</comment>
```